src/Microsoft.Azure.SignalR.Common/ServiceConnections/WebSocketConnectionContext.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Connections;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.Azure.SignalR.Connections.Client.Internal;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal class WebSocketConnectionContext : ConnectionContext
```

Comment:
Add a TODO: To implement features. It's not critical now since they aren't used by anything.